### PR TITLE
[Iga] Fix Lagrange support

### PR DESCRIPTION
**Pull request by tteschemacher**

### CODE REVIEW

**Comment by loumalouomega** [comment]
What was the problem?

File: applications/IgaApplication/custom_conditions/support_lagrange_condition.cpp [hunk 1]
```diff
@@ -96,44 +96,44 @@ namespace Kratos
...
-                                u[index + 1] = l_m[1] - displacement[1];
-                                u[index + 2] = l_m[2] - displacement[2];
-                                counter++;
-                            }
+                }
+                for (IndexType i = 0; i < number_of_nodes; i++) {
+                    for (IndexType n = 0; n < r_N.size1(); ++n) {
+                        if (r_N(n, i) > shape_function_tolerance) {
+                            const array_1d<double, 3>& r_l_m = r_geometry[i].FastGetSolutionStepValue(VECTOR_LAGRANGE_MULTIPLIER);
+                            IndexType index = 3 * (counter);
```
Comment (loumalouomega) [inline applications/IgaApplication/custom_conditions/support_lagrange_condition.cpp:127]: OK, Now I see the "bug"

**Review by loumalouomega — APPROVED** [review]
Okay

File: applications/IgaApplication/custom_conditions/support_lagrange_condition.cpp (hunk 1 above)
Comment (tteschemacher) [reply applications/IgaApplication/custom_conditions/support_lagrange_condition.cpp:127]: this was one part of it ;)

**Comment by tteschemacher** [comment]
> 
> 
> What was the problem?

**Pull request merged**